pom.xml

```diff
@@ -1623,6 +1622,10 @@
                 <include>src/main/java/**/*.java</include>
                 <include>src/test/java/**/*.java</include>
               </includes>
+              <excludes>
+                <exclude>src/main/java/org/apache/pinot/common/request/*.java</exclude>
+                <exclude>src/main/java/org/apache/pinot/common/response/ProcessingException.java</exclude>
+              </excludes>
```

Comment:
I insist. This is a bad practice. Luckily the path contains the name _common_ and we can infer this comes from pinot-common. Otherwise it would be difficult to understand where this comes from.

What if the path we want to exclude in pinot-common is also present in another project and in that other project we want to include it here? Doesn't make sense to move this particular pinot-common requirement to the parent pom.